src/components/List/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Icons } from "@storybook/components";
+import {
+  ListItemContentWrapper,
+  ListItemIndexWrapper,
+  ListItemWrapper,
+} from "./ListItem.styled";
+
+interface ListItemProps {
+  children: React.ReactNode;
+  nthItem: number;
+  isCompleted?: boolean;
+}
+
+export const ListItem = ({ children, nthItem, isCompleted }: ListItemProps) => {
+  return (
+    <ListItemWrapper>
+      <ListItemIndexWrapper
+        aria-label={isCompleted ? "complete" : "not complete"}
+        isCompleted={isCompleted}
+      >
+        {isCompleted ? <Icons icon="check" color="white" /> : nthItem}
```

Comment:
I believe you need to set the icon size to 10px

Reply:
how? Can you tell me?

Reply:
width={10} height={10} does the trick